### DISCUSSION TITLE
v0.7.0

### DISCUSSION THREAD

**Discussion by calavera** [aws/aws-lambda-rust-runtime#547] [General]
## What's Changed
* Check that `lambda_http`s inner service is ready before calling it by @dcormier in https://github.com/awslabs/aws-lambda-rust-runtime/pull/538
* Re-export external modules of publicly used types by @dcormier in https://github.com/awslabs/aws-lambda-rust-runtime/pull/542
* Decode query paramters for ALB event source by @bnusunny in https://github.com/awslabs/aws-lambda-rust-runtime/pull/545
* Update MSRV to 1.62 by @calavera in https://github.com/awslabs/aws-lambda-rust-runtime/pull/544
* Updated `aws_lambda_events` dependency by @dcormier in https://github.com/awslabs/aws-lambda-rust-runtime/pull/541
* Allow `Adapter` to be `.boxed()` by @dcormier in https://github.com/awslabs/aws-lambda-rust-runtime/pull/540

## New Contributors
* @dcormier made their first contribution in https://github.com/awslabs/aws-lambda-rust-runtime/pull/538

**Full Changelog**: https://github.com/awslabs/aws-lambda-rust-runtime/compare/v0.6.1...v0.7.0

<hr /><em>This discussion was created from the release <a href='https://github.com/awslabs/aws-lambda-rust-runtime/releases/tag/v0.7.0'>v0.7.0</a>.</em>

**Comment by dcormier** [comment]
Thanks for your help, @calavera! That was pretty satisfying.
